src/EditorFeatures/CSharpTest/InlineMethod/CSharpInlineMethodTests.cs

```diff
@@ -78,9 +149,51 @@ private class TestVerifier : CSharpCodeRefactoringVerifier<CSharpInlineMethodRef
             }
         }
 
+//         [Fact]
```

Comment:
??

Reply:
Ah, this is the test that I used to write the Test code. Forget to delete it.